Replicator/DBAccess.cc

```diff
@@ -110,6 +110,19 @@ namespace litecore { namespace repl {
         else
             return {};
     }
+    
+    
+    C4DocEnumerator* DBAccess::unresolvedDocsEnumerator(C4Error *outError) {
+        C4DocEnumerator* e;
+        use([&](C4Database *db) {
+            C4Error error = {};
+            C4EnumeratorOptions options = kC4DefaultEnumeratorOptions;
+            options.flags &= ~kC4IncludeNonConflicted;
```

Comment:
hoping this will include "conflicted + non-deleted", and "conflicted + deleted". some how when included the `kC4IncludeDeleted` it was not returning the non-deleted.

Reply:
`IncludeDeleted` will also return non-deleted.  It's just a flag to toggle inclusion, not to exclude the alternate.

Reply:
yes. correct.